empire.go

```diff
@@ -438,6 +448,10 @@ func (opts DeploymentsCreateOpts) Event() DeployEvent {
 	if opts.App != nil {
 		e.App = opts.App.Name
 	}
+
+	if opts.Environment != "" {
+		e.Environment = opts.Environment
```

Comment:
I don't think environment should be on the DeploymentsCreateOpts at all, you can just set the Environment on the even in the `Deploy` method:

```go
event.Environment = e.Environment
```

Which means that you don't need the GetEnvironment() method and the server doesn't need to know anything about passing it.